### PR TITLE
Modify Wazuh app and Wazuh plugin references

### PR DESCRIPTION
**Pull request by Tostti**
## Description
This PR modifies different references to `Wazuh app` and `Wazuh plugin` that were on the documentation. 

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [ ] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ Keep in mind that this repository is for the Wazuh documentation. There are some
 - If you have **problems, bugs or unexpected results** when using any of the Wazuh components, create the issue in its respective repository:
   - [Wazuh Core](https://github.com/wazuh/wazuh/issues)
   - [Wazuh API](https://github.com/wazuh/wazuh-api/issues)
-  - [Wazuh App](https://github.com/wazuh/wazuh-dashboard-plugins/issues)
-  - [Wazuh Splunk app](https://github.com/wazuh/wazuh-splunk/issues)
+  - [Wazuh Indexer](https://github.com/wazuh/wazuh-indexer/issues)
+  - [Wazuh Dashboard](https://github.com/wazuh/wazuh-dashboard/issues)
```
Comment (Desvelao) [inline CONTRIBUTING.md:20]: **nitpick**: The applications are called `Wazuh indexer` and `Wazuh dashboard`

File: source/user-manual/reference/daemons/wazuh-modulesd.rst [hunk 2]
```diff
@@ -23,7 +23,7 @@ The wazuh-modulesd program manages the Wazuh modules described below.
 
 .. topic:: CIS-CAT wodle
 
-  The CIS-CAT wodle allows you to run CIS policy scans visualizing the results of assessments in the Wazuh App. See the :doc:`CIS-CAT integration <../../capabilities/policy-monitoring/ciscat/ciscat>` for more information on this functionality.
+  The CIS-CAT wodle allows you to run CIS policy scans visualizing the results of assessments in the Wazuh Dashboard. See the :doc:`CIS-CAT integration <../../capabilities/policy-monitoring/ciscat/ciscat>` for more information on this functionality.
```
Comment (Desvelao) [inline source/user-manual/reference/daemons/wazuh-modulesd.rst:26]: **nitpick**: The application is called `Wazuh dashboard`

File: CONTRIBUTING.md (hunk 1 above)
Comment (Desvelao) [reply CONTRIBUTING.md:20]: Why are we removing the entries related to the plugins?

**Comment by Desvelao** [comment]
I found some references to Wazuh App I am not sure if they should be adapted or make sense:
- https://github.com/wazuh/wazuh-documentation/blob/fix-designations-about-wazuh-app-and-plugin/source/deployment-options/docker/wazuh-container.rst?plain=1#L47
- https://github.com/wazuh/wazuh-documentation/blob/fix-designations-about-wazuh-app-and-plugin/source/deployment-options/docker/wazuh-container.rst?plain=1#L147
- https://github.com/wazuh/wazuh-documentation/blob/fix-designations-about-wazuh-app-and-plugin/source/deployment-options/deploying-with-ansible/reference.rst?plain=1#L56

File: CONTRIBUTING.md (hunk 1 above)
Comment (Tostti) [reply CONTRIBUTING.md:20]: Our main repositories are going to be wazuh-indexer and wazuh-dashboard. As wazuh-dashboard represents the whole app, is going to be more straightforward for the users to report the bugs.The users may not be able to identify if a bug is happening due to the plugin or the dashboard structure. 
Additionally, we the repositories already exist and we are currently receiving issues through them

File: source/deployment-options/docker/wazuh-container.rst [hunk 3]
```diff
@@ -44,7 +44,7 @@ Single-node Deployment
         
       .. code-block:: yaml
         
-         # Wazuh App Copyright (C) 2021 Wazuh Inc. (License GPLv2)
+         # Wazuh dashboard Copyright (C) 2021 Wazuh Inc. (License GPLv2)
```
Comment (Desvelao) [inline source/deployment-options/docker/wazuh-container.rst:47]: This line is coming from a file of Docker deployment that is defined on the `wazuh-docker` repository. Regarding the context, it seems to be related to generates the certificates to using for Wazuh dashboard, Wazuh indexer and Wazuh server. It is not clear to me the usage of `App` term on this context.

https://github.com/wazuh/wazuh-docker/blob/v4.8.0-beta6/single-node/generate-indexer-certs.yml#L1

Note the year on the documentation is different to the `wazuh-docker` repository.

File: source/deployment-options/docker/wazuh-container.rst [hunk 4]
```diff
@@ -144,7 +144,7 @@ Multi-node deployment
       
       .. code-block:: yaml
       
-         # Wazuh App Copyright (C) 2021 Wazuh Inc. (License GPLv2)
+         # Wazuh dashboard Copyright (C) 2021 Wazuh Inc. (License GPLv2)
```
Comment (Desvelao) [inline source/deployment-options/docker/wazuh-container.rst:147]: This line is coming from a file of Docker deployment that is defined on the `wazuh-docker` repository. Regarding the context, it seems to be related to generates the certificates to using for Wazuh dashboard, Wazuh indexer and Wazuh server. It is not clear to me the usage of `App` term on this context.

https://github.com/wazuh/wazuh-docker/blob/v4.8.0-beta6/multi-node/generate-indexer-certs.yml#L1

Note the year on the documentation is different to the `wazuh-docker` repository.

Comment (asteriscos) [reply source/deployment-options/docker/wazuh-container.rst]: Fixed

File: source/deployment-options/docker/wazuh-container.rst (hunk 3 above)
Comment (asteriscos) [reply source/deployment-options/docker/wazuh-container.rst]: Fixed

**Review by javimed — APPROVED** [review]

**Pull request merged**